src/app/chat/create/page.tsx

```diff
@@ -0,0 +1,45 @@
+'use client';
+
+import { type ChangeEvent, useState } from 'react';
+import { Button, Header, Input } from '@/components/common';
+import styled from '@emotion/styled';
+
+const CreateChatPage = () => {
+  const [title, setTitle] = useState('');
+
+  const handleTitleInput = (e: ChangeEvent<HTMLInputElement>) => {
+    setTitle(e.target.value);
+  };
+
+  return (
+    <StyledCreateChatPage>
+      <Header title="채팅방 생성" onClose={() => {}} />
+
+      <Input
+        guideMessage="채팅방 주제를 입력해주세요"
+        label="채팅방 주제"
+        placeholder="최대 18글자"
+        onChange={handleTitleInput}
+      />
+
+      <Button
+        onClick={() => {}}
+        disabled={!title}
```

Comment:
```suggestion
        disabled={Boolean(title)}
```

제안입니다! 개인적으로 다음과 같이 Boolean으로 감싸주는게 가독성면에서 더 좋은것같아요!

Reply:
title이 없을 때 true여야 하니 이렇게 하는게 맞을 것 같아요!
```suggestion
        disabled={!Boolean(title)}
```